packages/accounts-base/accounts_client.js

```diff
@@ -150,6 +150,30 @@ export class AccountsClient extends AccountsCommon {
       });
   }
 
+  /**
+   * @summary Log out all clients logged in as the current user and logs the current user out as well.
+   * @locus Client
+   * @param {Function} [callback] Optional callback. Called with no arguments on success, or with a single `Error` argument on failure.
+   */
+  logoutAllClients(callback) {
+    this._loggingOut.set(true);
+
+    this.connection.applyAsync('logoutAllClients', [], {
+      // TODO[FIBERS]: Look this { wait: true } later.
```

Comment:
since Meteor 2.x is in [mantainece mode(eol december 2025)](https://forums.meteor.com/t/meteor-supported-versions-clarification-transition-timeline/63646), i'm not sure if we should add Fibers support here